rancher/dashboard
# Link to NeuVector in Cluster Explorer sidenav

This PR addresses https://github.com/rancher/dashboard/issues/5019 by adding a NeuVector side nav link and button when the NeuVector Helm chart is installed.

For reference, this PR is very similar to the one where a Longhorn link was added: https://github.com/rancher/dashboard/pull/1448

<img width="731" alt="Screen Shot 2022-03-03 at 3 50 51 PM" src="https://user-images.githubusercontent.com/20599230/156666217-2394a51c-d01c-4980-8762-445480ea3def.png">


To test this PR, I did this:

1. Go to Cluster Explorer.
2. Go to Apps & Marketplace > Repositories and click Create.
3. In the Target section, create a Git repository with the index URL of `https://github.com/selvamt94/charts.git` and the branch `neuvector`.
4. In Apps & Marketplace > Charts, install NeuVector from the new chart repo. When configuring the Helm chart values, go to the Container Runtime section, deselect Docker and select **k3s Containerd Runtime**. 
5. After NeuVector is installed, go to the left nav bar in Cluster Explorer and click NeuVector.
6. Click the external link and confirm that it goes to the NeuVector UI.

<img width="1029" alt="Screen Shot 2022-03-03 at 3 01 44 PM" src="https://user-images.githubusercontent.com/20599230/156661563-fd04b7d2-1027-4a46-903b-74c8108120f0.png">

I also uninstalled the NeuVector chart and confirmed that the link was removed.
